hooks/post/9-mongo_syspath.rb

```diff
@@ -0,0 +1,11 @@
+def install_syspaths
+  `rpm -qa | grep rh-mongodb34-syspaths`
+  if $?.success?
+    logger.info 'rh-mongodb34-syspaths already installed, skipping.'
```

Comment:
This should be `.debug` IMHO. Note this hook is also ran on certificate generation for capsules and you don't want to show this message. Running the hook itself doesn't hurt.

I noticed this from the Puppet upgrade hook while doing some unrelated testing so it's not the first instance where we made this mistake :)